### PR TITLE
Ungrok opengever.maintenance

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR ungroks opengever.maintenance

Related to https://github.com/4teamwork/opengever.core/issues/3205

### CODE REVIEW

**Comment by elioschmutz** [comment]
@phgross 👍

File: opengever/maintenance/browser/configure.zcml [hunk 1]
```diff
@@ -11,4 +11,11 @@
       permission="cmf.ManagePortal"
       />
 
+  <browser:page
+      for="*"
+      name="health-check"
+      class=".health_check.HealthCheckView"
+      permission="zope2.View"
```
Comment (phgross) [inline opengever/maintenance/browser/configure.zcml]: Need to be `zope2.Public`

File: opengever/maintenance/browser/configure.zcml [hunk 2]
```diff
@@ -40,4 +40,11 @@
       template="templates/lockmaintenanceview.pt"
       />
 
+  <browser:page
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      name="mnt-warmup"
+      class=".warmup.WarmupView"
+      permission="zope2.View"
```
Comment (phgross) [inline opengever/maintenance/browser/configure.zcml]: `zope.Public`

**Comment by elioschmutz** [comment]
@phgross  done 👍

**Pull request merged**